tests/improved-sanity-test/main.yml

```diff
@@ -878,6 +878,15 @@
       tags:
         - rpm_ostree_cleanup_all
 
+    # remove any docker artifacts
+    - role: docker_remove_all
+
```

Comment:
We seem to have tags for all our roles in this playbook.  Lets add them here to be consistent.